src/Chapter1.hs

```diff
@@ -638,11 +657,15 @@ Implement a function that returns the first digit of a given number.
 5
 
 You need to use recursion in this task. Feel free to return to it later, if you
-aren't ready for this boss yet!
--}
+aren't ready for this boss yet! :r
 
-firstDigit n = error "firstDigit: Not implemented!"
+-}
 
+firstDigit :: Int -> Int
+firstDigit n 
+  | n < 10 = n
+  | otherwise = repeat
+  where repeat = firstDigit $ n `div` 10
```

Comment:
`repeat` is a standard function, so I would suggest to rename it to avoid name conflicts.